src/instruments/src/MCDU/Pages/FMGC/RadNav/Radnav.tsx

```diff
@@ -0,0 +1,334 @@
+    };
+    return (
+        <RowHolder columns={2}>
+            <LineHolder columnPosition={1}>
+                <LabelField lineSide={lineSides.left} value="ADF1/FREQ" color={lineColors.white} />
+                <StringInputField
+                    lineSide={lineSides.left}
+                    defaultValue={adfone}
+                    nullValue="[  ]/[   .]"
+                    color={adfone !== undefined ? lineColors.cyan : lineColors.cyan}
+                    lsk={LINESELECT_KEYS.L5}
+                    selectedCallback={(value) => setNewValue(value, 1)}
+                    selectedValidation={((value) => validateEntry(value, 1))}
+                    size={lineSizes.regular}
+                />
+            </LineHolder>
+            <LineHolder columnPosition={2}>
+                <LabelField lineSide={lineSides.right} value="FREQ/ADF2" color={lineColors.white} />
+                <StringInputField
+                    lineSide={lineSides.right}
+                    defaultValue={adftwo}
+                    nullValue="[    .]/[ ]"
+                    color={adftwo !== undefined ? lineColors.cyan : lineColors.cyan}
+                    lsk={LINESELECT_KEYS.R5}
+                    selectedCallback={(value) => setNewValue(value, 2)}
+                    selectedValidation={((value) => validateEntry(value, 2))}
+                    size={lineSizes.regular}
+                />
+            </LineHolder>
+        </RowHolder>
+
+    );
+};
+
+const Adfbfo: React.FC = () => (
+    <RowHolder columns={2}>
+        <LineHolder columnPosition={1}>
+            <EmptyLine />
+            <Field
+                lineSide={lineSides.left}
+                value="<-ADF1 BFO"
+                size={lineSizes.regular}
+                color={lineColors.inop}
+            />
+        </LineHolder>
+        <LineHolder columnPosition={2}>
+            <EmptyLine />
+            <Field
+                lineSide={lineSides.right}
+                value="ADF2 BFO->"
+                size={lineSizes.regular}
+                color={lineColors.inop}
+            />
+        </LineHolder>
+    </RowHolder>
+);
+
+const RadnavPage: React.FC = () => {
+    const dispatch = useMCDUDispatch();
+    const addMessage = (msg: ScratchpadMessage) => {
+        dispatch(scratchpadActions.addScratchpadMessage(msg));
+    };
+
+    const setTitlebar = ((msg: string) => {
+        dispatch(titlebarActions.setTitleBarText(msg));
+    });
+    const clearMessage = () => {
+        dispatch(scratchpadActions.clearScratchpad());
+    };
+    useEffect(() => {
+        setTitlebar(' RADIO NAV ');
+    }, []);
+    return (
+        <>
+            <Vor addMessage={addMessage} clearMessage={clearMessage} />
+            <Crs clearMessage={clearMessage} />
+            <Ls addMessage={addMessage} clearMessage={clearMessage} />
+            <Lscrs clearMessage={clearMessage} />
+            <Adf addMessage={addMessage} clearScratchpad={clearMessage} />
```

Comment:
All interactive components automatically clear the scratchpad, you don't need to handle that. https://github.com/flybywiresim/a32nx/blob/1bdbc95860f8b482c0928e3e349256d74aa6ef84/src/instruments/src/MCDU/Components/Fields/Interactive/StringInputField.tsx#L55-L62